Sources/RxAlamofire.swift

```diff
@@ -775,11 +775,11 @@ extension Reactive where Base: DataRequest {
                 case .success(let result):
                     if let httpResponse = packedResponse.response {
                         observer.on(.next(httpResponse, result))
+                        observer.on(.completed)
                     }
                     else {
                         observer.on(.error(RxAlamofireUnknownError))
                     }
-                    observer.on(.completed)
```

Comment:
Actually the [Alamofire 4.0 Migration Guide](https://github.com/Alamofire/Alamofire/blob/master/Documentation/Alamofire%204.0%20Migration%20Guide.md#benefits-of-upgrading) claims 

> New Request Types: include DataRequest, DownloadRequest, UploadRequest and **StreamRequest** that implement specialized progress, validation and serialization APIs and behaviors per Request type.

Would that mean DataRequest doesn't work with streamed request here? If yes this fix should be valid.